plugins/tokenizers/mecab.c

```diff
@@ -1063,6 +1064,34 @@ mecab_next_default_format(grn_ctx *ctx,
   }
 }
 
+int
+mecab_wakati_delimiter(const char *str, grn_encoding encoding)
+{
+  const unsigned char *s = (const unsigned char *)str;
+  switch (s[0]) {
+  case '\t':
+    return 1;
+  case 0x81:
+    if (encoding == GRN_ENC_SJIS && s[1] == 0x40) {
+      return 2;
+    }
+    break;
+  case 0xA1:
+    if (encoding == GRN_ENC_EUC_JP && s[1] == 0xA1) {
+      return 2;
+    }
+    break;
+  case 0xE3:
+    if (encoding == GRN_ENC_UTF8 && s[1] == 0x80 && s[2] == 0x80) {
+      return 3;
+    }
+    break;
```

Comment:
```suggestion
```

This code is for full-width a space, so it should not be necessary.

Reply:
We need this code to support the following case although the function name might be misleading.
- https://github.com/groonga/groonga/blob/main/test/command/suite/tokenizers/mecab/full_width_space/last.test

Reply:
It seems that we just need to remove leading/trailing spaces for each tokens.